static/espeasy.js

```diff
@@ -1,11 +1,41 @@
 // CodeMirror, copyright (c) by Marijn Haverbeke and others
 // Distributed under an MIT license: https://codemirror.net/LICENSE
+
+var commonAtoms = ["And", "Or"];
+var commonKeywords = ["If", "Else", "Elseif", "Endif"];
+var commonCommands = ["AcessInfo", "Background", "Build", "ClearAccessBlock", "ClearRTCam", "Config", "ControllerDisable",
+  "ControllerEnable", "DateTime", "Debug", "DeepSleep", "DNS", "DST", "EraseSDKwifi", "ExecuteRules", "Gateway", "I2Cscanner",
+  "IP", "Let", "Load", "LogEntry", "LogPortStatus", "LoopTimerSet", "LoopTimerSet_ms", "MemInfo", "MemInfoDetail", "Name", "Password", "Publish",
+  "Reboot", "Reset", "ResetFlashWriteCounter", "Save", "SendTo", "SendToHTTP", "SendToUDP", "Settings", "Subnet", "Subscribe", "TaskClear", "TaskClearAll",
+  "TaskDisable", "TaskEnable", "TaskRun", "TaskValueSet", "TaskValueSetAndRun", "TimerPause", "TimerResume", "TimerSet", "TimerSet_ms", "TimeZone",
+  "UdpPort", "UdpTest", "Unit", "UseNTP", "WdConfig", "WdRead", "WifiAPkey", "WifiAllowAP", "WifiAPMode", "WifiConnect", "WifiDisconnect", "WifiKey",
+  "WifiKey", "WifiScan", "WifiSSID", "WifiSSID2", "WifiSTAMode",
+  "Event", "AsyncEvent", "/control?cmd",
```

Comment:
I'm not sure if "/control?cmd" should be a suggestion or keyword, IMHO it shouldn't be in here.

Reply:
Thanks.. i copied these words blindly from the notepad++ version.. i also asked myself why "/control?cmd"  is in the list 🤷🏻‍♂️

Reply:
And the second "WifiKey" instance should probably read "WiFiKey2" (spelling the word WiFi must be difficult, for whoever created the original list 😛)

Reply:
> > I'm not sure if "/control?cmd" should be a suggestion or keyword, IMHO it shouldn't be in here.

i am going through all the plugins right now...
thats why: https://espeasy.readthedocs.io/en/latest/Plugin/P091.html#p091-page

Reply:
Well, that's just *one* way of activating that command, not **the** way, nor the generic way 😉